packages/insomnia-app/app/ui/components/modals/request-settings-modal.js

```diff
@@ -133,10 +142,16 @@ class RequestSettingsModal extends React.PureComponent<Props, State> {
       return;
     }
 
-    await models.request.update(request, {
+    const patch = {
       metaSortKey: -1e9, // Move to top of sort order
       parentId: activeWorkspaceIdToCopyTo,
-    });
+    };
+
+    // TODO: if gRPC, we should also copy the protofile to the destination workspace - INS-267
+
+    isGrpcRequest(request)
+      ? await models.grpcRequest.update(request, patch)
+      : await models.request.update(request, patch);
```

Comment:
It's possible to use the following syntax (here and elsewhere) but I chose to keep it explicit. We can do a sweep after this PR and update if need be, but it is safer to keep explicit.

```suggestion
    await models.mustGetModel(request.type).update(request, patch);
```